data/vibe/data/serialization.d

```diff
@@ -1487,6 +1488,22 @@ private template getExpandedFieldsData(T, FIELDS...)
 	alias getExpandedFieldsData = staticMap!(processSubfield, FIELDS);
 }
 
+/// Policy for serializing arrays using base64 encoding
+public class Base64ArraySerPolicy (R) if (isArray!R && is(ElementType!R : ubyte))
+{
+	public static string toRepresentation (in R data) @safe pure
+	{
+		import std.base64 : Base64;
+		return Base64.encode(data);
+	}
+
+	public static ubyte[] fromRepresentation (in string data) @safe pure
+	{
+		import std.base64 : Base64;
+		return Base64.decode(data);
+	}
+}
```

Comment:
So, should we try writing to a sink / input range here ? :D

Reply:
I think if we want to have the policies support sink type serialization, then it should be a separate ticket. Polices were added to vibe.d long time ago.